source/_integrations/kostal_plenticore.markdown

```diff
@@ -96,6 +96,18 @@ The following sensors are available in the library:
 | Energy Yield Month      | kWh  | Energy yield of the current month. |
 | Energy Yield Year       | kWh  | Energy yield of the current year. |
 | Energy Yield Total      | kWh  | Energy yield total. |
+| Energy Discharge to Grid Day    | kwH  | Energy discharged to the Grid of the current day. |
```

Comment:
Those units of measurements are not correct, `kwH` -> `kWh`

Reply:
woopsie, fixed